bin/raptor

```diff
@@ -94,17 +95,22 @@ parent
     transform: utils.toInt,
     callback: utils.validate('connectionTimeout', utils.validator.isInt)
   })
+  .option('logcat', {
```

Comment:
I can see your point, but I'm not sure when `--logcat` would be used other than with `test`. `--metrics` is used in `test` and `submit`.